docs/clients/athena.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: client
+category: clients
+name: Athena
+package: async-aws/athena
+---
+
+## Usage
+
+### List Databases
+
+```php
+use AsyncAws\Athena\AthenaClient;
+use AsyncAws\Athena\Input\ListDatabasesInput;
+
+$athena = new AthenaClient();
+
+$result = $athena->listDatabases(new ListDatabasesInput([
+    'CatalogName' => 'my_catalog'
+    'NextToken' => 'next-1045'
+    'MaxResults' => 20
+]));
+
+$databaseList = $result->getDatabaseList();
+foreach ($result->getDatabaseList() as $database) {
+    echo 'Database name : ' . $database->getName();
+    echo 'Database description : ' . $database->getDescription();
+    echo 'Database parameter : '.PHP_EOL;
+    print_r($database->getParameters());
+}
+
+echo 'Next token : ' . $result->getNextToken(). "\n";
```

Comment:
to remove: Because async-aws take care of pagination, and nextToken will always be null ;)